.github/workflows/docker.yml

```diff
@@ -110,3 +111,47 @@ jobs:
           file: docker/blockchain/Dockerfile
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.meta.outputs.tags }}
+
+  ghcr-publish:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/')
```

Comment:
suggestion: Let's get rid of this `if` line so that it runs the same as the other docker jobs.